manual/src/threelang/tfchain/tfchain.md

```diff
@@ -0,0 +1,237 @@
+# TF Chain Actions
+In a simple format you can do almost all the extrinsic and queries on the chian. List of the namespaces
+
+## Name spaces and their actions
+- [Account](#account)
+- [Balance](#balance)
+- [Twins](#twins)
+- [Nodes](#nodes)
+- [Farms](#farms)
+- [Contracts](#contracts)
+- [Service Contract](#service-contract)
+- [Metadata](#metadata)
+
+
+## Account
+- Create new account on the chain by Generating mnemonic
+    ```md
+    !!tfchain.account.create
+        network:devnet 
+    ```
+    - `network`: is the tfchain network, should be one of (mainnet, testnet, qanet, devnet)
+- Load the account to do chain extrinsic 
+    ```md
+    !!tfchain.client.load
+        network:devnet
+        mnemonic:\'secret words as your mnemonic\'
```

Comment:
I'm not sure but I think you can leave out the \'. The actions parser will know it's a string I think. But make sure to double check.